.gitignore

```diff
@@ -50,3 +50,5 @@ bazel-arcs
 **/.aswb/**
 .gradle
 *.iml
+.*
```

Comment:
I don't think this is correct. There's files like .clang-format, .travis.yml, .eslintignore, etc that are important?

Reply:
I changed to to more detailed ignores. Good to know some of the hidden files are useful, thanks.